site/docs/v1/tech/apis/_webauthn-register-complete-request-body.adoc

```diff
@@ -0,0 +1,40 @@
+==== Request Body
+
+The [field]#credential# in the request body contains data returned by the WebAuthn client API call. The [field]#credential.response.attestationObject# and [field]#credential.response.clientDataJSON# fields are returned as ``ArrayBuffer``s by the WebAuthn client API and must be converted to base64url-encoded strings for the FusionAuth API request.
+
+[.api]
+[field]#credential.clientExtensionResults.credProps.rk# [type]#[Boolean]# [optional]#Optional#::
+If `true`, the new passkey is "client-side discoverable," meaning that it can be used in authentication ceremonies without first identifying the user (e.g. by requiring the user to complete a form field). If `false`, or the [field]#credential.clientExtensionResults.credProps# field is omitted, the passkey should be treated as if it were _not_ client-side discoverable. FusionAuth does not currently support discoverable credentials (sometimes referred to as "resident keys").
```

Comment:
The value on this response is set by the authenticator (or possibly the browser). I don't remember why I added it. Perhaps because I thought we would have discoverable passkeys supported for the initial release?

Either way, I'm using it to record whether a passkey is discoverable in the database. It could be useful once the feature is supported.

Reply:
Roger that.